like/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+from user.models import User
+from ingredient.models import Ingredient
+
+class Like(models.Model):
+    date = models.DateField()
```

Comment:
위를 참고해주셔서 created_at으로 수정해주세요~

```suggestion
    created_at = ...
```

Reply:
created_at 변경후 datefield를 datetimefield로 수정완료했습니다.